docs/articles/documentation/reference/testcafe-api/global/createtestcafe.md

```diff
@@ -10,7 +10,7 @@ redirect_from:
 Creates a [TestCafe](../testcafe/README.md) server instance.
 
 ```text
-async createTestCafe([hostname], [port1], [port2], [sslOptions], [developmentMode]) → Promise<TestCafe>
+async createTestCafe([hostname], [port1], [port2], [sslOptions], [developmentMode], [retryTestPages], [cache]) → Promise<TestCafe>
```

Comment:
I propose to rewrite it using the more preferred syntax since we change it.

The parameters can be specifed as a single object:
```js
async createTestCafe (options) -> Promise<TestCafe>

where options is
{
   hostname: string,
   port1: number,
   port2: number,
   sslOptions: SslOptions,
  developmentMode: boolean,
  retryTestPages: boolean,
  cache: boolean



}
```